code/src/index.css

```diff
@@ -5,9 +5,129 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+
+}
+
+.artist, .album, .albumCover {
+  grid-column: 1/span 2;
+  justify-self: center;
+  margin: 0 auto;
+
+}
+
+.artist span {
+  font-family: 'Helvetica', sans-serif;
+  font-size: 14px;
+  color: #a0a0a0;
+  text-decoration: none;
+  justify-self: start;
+}
+
+.artist a, .album a {
+  text-decoration: none;
+}
+
+.artist a:hover {
+  text-decoration: underline;
+  color: #a0a0a0;
+}
+
+.album span {
+  font-family: 'Helvetica', sans-serif;
+  font-size: 14px;
+  color: white;
+  text-decoration: none;
+}
+
+.album a:hover {
+  text-decoration: underline;
+  color: #ffffff;
+}
+
+.image-container {
+  width: 100%;
+  position: relative
+}
+
+.albumImage {
+  width: 100%;
+  object-fit: contain;
+  justify-self: center;
+}
+
+.image-overlay {
+  top: 0;
+  z-index: 1;
```

Comment:
Nice use of the `z-index` here; I guess you were trying to make the album cover fade not obstruct the brightness of the icons? However, I think with` 'image-overlay' `you have targeted only your icons, so you may need to add another z-index to `‘image-container’`.